src/accelerate/components/List.tsx

```diff
@@ -1,31 +1,36 @@
 import React, { useState } from 'react';
 import { useSelect } from '@wordpress/data';
 import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalRadioGroup as RadioGroup, __experimentalRadio as Radio } from '@wordpress/components';
 import { Pagination } from 'react-pagination-bar';
 
+import { Dropdown, Button, MenuGroup, MenuItem } from '@wordpress/components';
+
+import { periods } from '../../data/periods';
 import { compactMetric, Duration, getConversionRateLift, InitialData, Post, State } from '../../util';
 
 import './Dashboard.scss';
 
 let timer: ReturnType<typeof setTimeout> | undefined;
 
 type Props = {
-	period: Duration,
 	postTypes: InitialData['postTypes'],
 	user: InitialData['user'],
 };
 
-export default function List( props: Props ) {
+export default function List ( props: Props ) {
 	// Filters.
-	const [ search, setSearch ] = useState<string>( '' );
-	const [ page, setPage ] = useState<number>( 1 );
-	const [ type, setType ] = useState<string|null>( null );
-	const [ user, setUser ] = useState<number|null>( null );
+	const [customFilter, setCustomFilter] = useState<string>( 'all' );
```

Comment:
Not sure what happened with the spacing here, not urgent but maybe check auto formatting settings.

Reply:
Yeah, sorry about that. Will adjust my editor settings.